packages/server/app/Controllers/Http/MtvRoomsHttpController.test.ts

```diff
@@ -52,6 +52,19 @@ test.group('MtvRoom Search Engine', (group) => {
         await request.post('/search/rooms').send(body).expect(401);
     });
 
+    test('Requires authentication', async () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏I think the test should be named _Requires authorization_.